eviatar/enableFeatures_createUsers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"
+        Install-WindowsFeature -Name $Feature -IncludeAllSubFeature -IncludeManagementTools -Verbose
+    }
+}
+
+function Create-LocalUsers {
+    param (
+        [Parameter(Mandatory = $true)]
+        [PSObject[]]$Users

```

Comment:
Are you sure that PSObject[] is the best type top use here?

Reply:
You right, I am dealing with structured user data, [PSCustomObject[]] or [Hashtable[]] might be better options, as they provide more structure and flexibility.